### DISCUSSION TITLE
Basic stats for all!

### DISCUSSION THREAD

**Discussion by chinchang** [cssbattle/forum#255] [Announcements]
Hola Players!!

As some of you might already know that we had a PRO feature called "My Analytics" where you could see some basic stats about your played targets. We recently did an overhaul of that page to give you some powerful advanced analytics on your targets!
But there was never a way for everyone to see what targets they had played and what not.

So today we are opening /me page to all players, with basic stats!!!
You can go to https://cssbattle.dev/me and see your basic target stats! And if you want to see some ["Advanced analytics"](https://cssbattle.dev/blog/advanced-analytics), you can always [Upgrade to PRO](https://cssbattle.dev/pro).

Cheers!

**Comment by rafehimself** [comment]
![image](https://user-images.githubusercontent.com/58576132/164885540-6f388216-ee37-4a73-8ac2-e553c3c6e962.png)
bugs,
not working, loading forever

**Reply by chinchang** [reply]
Ouch....fixed now. Should be working. Sorry about that!

**Reply by rafehimself** [reply]
no worries 😃 , happens
